apps/music/js/ui/title_bar.js

```diff
@@ -61,13 +65,18 @@ var TitleBar = {
 
   showScanProgress: function(info) {
     this.scanProgress.classList.remove('hidden');
+    this.scanSpinner.classList.remove('hidden');
```

Comment:
I'm not sure what the bug number is for a bug fix that would allow us to remove this line. Fixing 1166500 won't necessarily allow this to be removed. And also, removing this line without editing the index.html file won't work either. So adding a comment here might just make things more confusing than they already are.